pkg/workflows/wasm/host/module.go

```diff
@@ -110,7 +110,15 @@ type ModuleConfig struct {
 	Determinism *DeterminismConfig
 }
 
-type Module struct {
+type Module interface {
+	Start()
+	Close()
+	IsLegacyDAG() bool
+	Run(ctx context.Context, request *wasmpb.Request) (*wasmpb.Response, error)
```

Comment:
_nit_ docstring for the interface methods

Reply:
done